flink/json-jobmanager.yml

```diff
@@ -0,0 +1,224 @@
+    metric:
+      - name: jobmanager.direct.max
+        source: value
+        type: long_gauge
+        unit: bytes
+        label: direct used
+        description: The total capacity of all buffers in the direct buffer pool
+
+  - name: jobmanager_direct_used
+    metricNamespace: flink
+    path: $.[?(@.id=Status.JVM.Memory.Direct.MemoryUsed)]
+    metric:
+      - name: jobmanager.direct.used
+        source: $.[?(@.id=Status.JVM.Memory.Direct.MemoryUsed)].value
+        type: long_gauge
+        unit: bytes
+        label: direct used
+        description: The amount of memory used by the JVM for the direct buffer pool
+
+  - name: jobmanager_direct_count
+    metricNamespace: flink
+    path: $.[?(@.id=Status.JVM.Memory.Direct.Count)]
+    metric:
+      - name: jobmanager.direct.count
+        source: value
+        type: long_gauge
+        label: direct count
+        description: The number of buffers in the direct buffer pool
+
+  - name: jobmanager_mapped_max
+    metricNamespace: flink
+    path: $.[?(@.id=Status.JVM.Memory.Mapped.TotalCapacity)]
+    metric:
+      - name: jobmanager.mapped.max
+        source: value
+        type: long_gauge
+        unit: bytes
+        label: mapped count
+        description: The number of buffers in the mapped buffer pool
+
+  - name: jobmanager_mapped_used
+    metricNamespace: flink
+    path: $.[?(@.id=Status.JVM.Memory.Mapped.MemoryUsed)]
+    metric:
+      - name: jobmanager.mapped.used
+        source: value
+        type: long_gauge
+        unit: bytes
+        label: mapped used
+        description: The amount of memory used by the JVM for the mapped buffer pool
+
+  - name: jobmanager_mapped_count
+    metricNamespace: flink
+    path: $.[?(@.id=Status.JVM.Memory.Mapped.Count)]
+    metric:
+      - name: jobmanager.mapped.count
+        source: value
+        type: long_gauge
+        label: mapped count
+        description: The number of buffers in the mapped buffer pool
+
+  - name: jobmanager_threads_count
+    metricNamespace: flink
+    path: $.[?(@.id=Status.JVM.Threads.Count)]
+    metric:
+      - name: jobmanager.threads.count
+        source: value
+        type: long_gauge
+        label: The total number of live threads
+
+  - name: jobmanager_cpu_time
+    metricNamespace: flink
+    path: $.[?(@.id=Status.JVM.CPU.Time)]
+    metric:
+      - name: jobmanager.cpu.time
+        source: value
+        type: long_gauge
+        label: cpu time
+        description: The CPU time used by the JVM
+
+  - name: jobmanager_cpu_load
+    metricNamespace: flink
+    path: $.[?(@.id=Status.JVM.CPU.Load)]
+    metric:
+      - name: jobmanager.cpu.load
+        source: value
+        type: long_gauge
+        label: cpu load
+        description: The recent CPU usage of the JVM
+
+  - name: jobmanager_class_loaded
+    metricNamespace: flink
+    path: $.[?(@.id=Status.JVM.ClassLoader.ClassesLoaded)]
+    metric:
+      - name: jobmanager.class.loade
```

Comment:
`jobmanager.classes.loaded`. Use `classes` plural in other similar metrics.

Reply:
Fixed